graphql/schema.graphql

```diff
@@ -1126,6 +1126,11 @@ type GroupedProjects {
   projects: [Project!]!
 }
 
+type Permissions {
+  userId: String!
+  canCreateProject: Boolean!
```

Comment:
Now that we have this setup, we should also move canModifyAnnotation to here, either in this pr or in another ticket.

Reply:
and probably also CanOverrideDependencies

Reply:
@malikchaya2 could you make a ticket to handle this in other ticket? (not sure if those are big enough to be separate tickets or if they should be handled in the same one)

Reply:
https://jira.mongodb.org/browse/EVG-16500